.github/workflows/cd-infrastructure.yml

```diff
@@ -9,13 +9,14 @@ on:
     branches:
       - main
     paths: ['.github/workflows/**','infrastructure/**']
```

Comment:
I think we might need to refine the first path in the array now to only .github/workflows/cd-infra.yml and reusable terraform since there are many workflows now, we do not want to overlap when doing a PR or merge